### PR TITLE
[fdb] Decrease the dummy mac count to improve the running time

### PR DESCRIPTION
**Pull request by kevinskwang**
What is the motivation for this PR?
In FDB test case, it will run over 90 mins on the topo t0-116 which has doubled the ports number than standard t0. It makes ptf too busy to handle all the packets. 

How did you do it?
To decrease the test case running time and mitigate the ptf performance bottleneck, we decrease the dummy mac account from 10 to 2 on topo-116.

How did you verify/test it?
Run test_fdb.py on topo-116, each subcase will be finished within 10 mins.

Signed-off-by: Kevin(Shengkai) Wang <shengkaiwang@microsoft.com>

### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911

### CODE REVIEW

File: tests/fdb/test_fdb.py [hunk 1]
```diff
@@ -148,10 +160,13 @@ def send_recv_eth(ptfadapter, source_ports, source_mac, dest_ports, dest_mac, sr
     logger.debug('send packet src port {} smac: {} dmac: {} vlan: {} verifying on dst port {}'.format(
         source_ports, source_mac, dest_mac, src_vlan, dest_ports))
     testutils.send(ptfadapter, source_ports[0], pkt)
-    testutils.verify_packet_any_port(ptfadapter, exp_pkt, dest_ports, timeout=FDB_WAIT_EXPECTED_PACKET_TIMEOUT)
+    if len(dest_ports) == 1:
+        testutils.verify_packet(ptfadapter, exp_pkt, dest_ports[0], timeout=FDB_WAIT_EXPECTED_PACKET_TIMEOUT)
```
Comment (ganglyu) [inline tests/fdb/test_fdb.py:164]: Need to guarantee other port can't receive exp_pkt.

File: tests/fdb/test_fdb.py [hunk 2]
```diff
@@ -306,7 +322,7 @@ def test_fdb(ansible_adhoc, ptfadapter, duthosts, rand_one_dut_hostname, ptfhost
 
     vlan_member_count = sum([ len(members) for members in vlan_table.values() ])
 
-    fdb = setup_fdb(ptfadapter, vlan_table, router_mac, pkt_type)
+    fdb = setup_fdb(ptfadapter, vlan_table, router_mac, pkt_type, configured_dummay_mac_count)
```
Comment (ganglyu) [inline tests/fdb/test_fdb.py:325]: get_dummy_mac_count can be used here

File: tests/fdb/test_fdb.py [hunk 3]
```diff
@@ -25,13 +25,25 @@
...
 FDB_WAIT_EXPECTED_PACKET_TIMEOUT = 5
 PKT_TYPES = ["ethernet", "arp_request", "arp_reply", "cleanup"]
 
 logger = logging.getLogger(__name__)
 
+@pytest.fixture(scope="module")
+def get_dummay_mac_count(tbinfo):
+    REQUIRED_TOPO = ["t0-116"]
+    if tbinfo["topo"]["name"] in REQUIRED_TOPO:
+        # To reduce the case running time and avoid mac age issue
```
Comment (ganglyu) [inline tests/fdb/test_fdb.py:40]: This case has disabled fdb age.
https://github.com/Azure/sonic-mgmt/blob/master/tests/fdb/test_fdb.py#L22

File: tests/fdb/test_fdb.py (hunk 1 above)
Comment (kevinskwang) [reply tests/fdb/test_fdb.py:164]: Agree~

File: tests/fdb/test_fdb.py (hunk 2 above)
Comment (kevinskwang) [reply tests/fdb/test_fdb.py:325]: it is equal

File: tests/fdb/test_fdb.py (hunk 3 above)
Comment (kevinskwang) [reply tests/fdb/test_fdb.py:40]: Yes, should be not related with fdb age, changed the commit message~

**Review by ganglyu — APPROVED** [review]
lgtm

**Review by wangxin — APPROVED** [review]

**Pull request merged**